src/pages/my_registrations/MyRegistrations.tsx

```diff
@@ -54,16 +95,31 @@ export const MyRegistrations = ({ selectedUnpublished, selectedPublished }: MyRe
           <ListSkeleton minWidth={100} maxWidth={100} height={100} />
         ) : (
           <>
-            <Typography gutterBottom variant="h2">
-              {t('common.result_registrations')}
-            </Typography>
+            <Box sx={{ display: 'flex', alignItems: 'center', justifyContent: 'space-between' }}>
+              <Typography gutterBottom variant="h2">
+                {t('common.result_registrations')}
+              </Typography>
+              <Button variant="contained" onClick={deleteAllOnClick}>
+                {t('my_page.registrations.delete_all_draft_registrations')}
+              </Button>
+            </Box>
             <MyRegistrationsList
-              registrations={filteredRegistrations.length > 0 ? filteredRegistrations : registrations}
+              registrations={filteredRegistrations.length > 0 ? filteredRegistrations : []}
```

Comment:
Her gjelder sikkert samme som på forrige PR? Om det ikke er en bevisst grunn til å sette tomt array her da, men ser ikke hva det evt skal være
```suggestion
              registrations={filteredRegistrations}
```